test/e2e_test.sh

```diff
@@ -59,10 +64,15 @@ export FULCIO_METRICS_PORT=2113
 export FULCIO_CONFIG=/tmp/fulcio-config.json
 for repo in rekor fulcio; do
     pushd $repo
-    docker-compose up -d
+    if [ "$repo" == "fulcio" ]; then
+       yq -i e '.networks={"default":{ "name":"fulcio_default","external":true }}' docker-compose.yml
+       yq -i e '.services.fulcio-server.networks=["default"]' docker-compose.yml
+       tail docker-compose.yml
+    fi
```

Comment:
FYI I think all this is unnecessary if you change line 27 to use label `com.docker.compose.network=default`